helm/templates/deployment.yaml

```diff
@@ -58,6 +58,10 @@ spec:
               name: root-ca
               subPath: {{ quote .Values.global.ca.key }}
             {{- end }}
+            {{- if $db.sslEnabled  }}
+            - name: db-certs
+              mountPath: /mnt/db/certs
```

Comment:
check how to remove full dir path duplication

Reply:
moved to value